repose-aggregator/components/filters/body-extractor-to-header-filter/src/test/scala/org/openrepose/filters/bodyextractortoheader/BodyExtractorToHeaderSchemaTest.scala

```diff
@@ -20,17 +20,20 @@
 package org.openrepose.filters.bodyextractortoheader
 
 import org.junit.runner.RunWith
+import org.mockito.Mockito._
 import org.openrepose.commons.test.ConfigValidator
+import org.openrepose.core.services.config.ConfigurationService
 import org.scalatest.junit.JUnitRunner
 import org.scalatest.{FunSpec, Matchers}
 
 @RunWith(classOf[JUnitRunner])
 class BodyExtractorToHeaderSchemaTest extends FunSpec with Matchers {
-  val validator = ConfigValidator("/META-INF/schema/config/body-extractor-to-header.xsd")
+  val filter: BodyExtractorToHeaderFilter = new BodyExtractorToHeaderFilter(mock(classOf[ConfigurationService]))
```

Comment:
I don't love that you have to instantiate the filter (and mock a configuration service) just to get the paths you want.

Can we make those variables static? In Scala, that would be moving them to a companion object. I'm not sure if you can override them in the companion object, since you're using the abstract filter, but if not, you should be able to define them in a companion object, and override them in the class by pointing the references to the companion object.